images/microOS/config.sh

```diff
@@ -58,125 +56,31 @@ suseImportBuildKey
-cat >/etc/dracut.conf.d/50-microos-growfs.conf <<"EOF"
-install_items+=" /usr/lib/systemd/systemd-growfs "
-EOF
-
-#======================================
-# Disable recommends on virtual images (keep hardware supplements, see bsc#1089498)
-#--------------------------------------
-sed -i 's/.*solver.onlyRequires.*/solver.onlyRequires = true/g' /etc/zypp/zypp.conf
-
-#======================================
-# Disable installing documentation
-#--------------------------------------
-sed -i 's/.*rpm.install.excludedocs.*/rpm.install.excludedocs = yes/g' /etc/zypp/zypp.conf
-
-#======================================
-# Configure Pine64 specifics
-#--------------------------------------
-if [[ "$kiwi_profiles" == *"Pine64" ]]; then
-    echo 'add_drivers+=" fixed sunxi-mmc axp20x-regulator axp20x-rsb "' > /etc/dracut.conf.d/sunxi_modules.conf
-fi
-
-#======================================
-# Configure Raspberry Pi specifics, unless done by raspberrypi-firmware already (on TW)
-#--------------------------------------
-if [[ "$kiwi_profiles" == *"RaspberryPi"* ]] && ! [[ -e /usr/lib/dracut/dracut.conf.d/raspberrypi_modules.conf ]]; then
-	# Add necessary kernel modules to initrd (will disappear with bsc#1084272)
-	echo 'add_drivers+=" bcm2835_dma dwc2 "' > /etc/dracut.conf.d/raspberrypi_modules.conf
-
-	# Add necessary kernel modules to initrd (will disappear with boo#1162669)
-	echo 'add_drivers+=" pcie-brcmstb "' >> /etc/dracut.conf.d/raspberrypi_modules.conf
-
-	# Work around network issues
-  	cat > /etc/modprobe.d/50-rpi3.conf <<-EOF
-		# Prevent too many page allocations (bsc#1012449)
-		options smsc95xx turbo_mode=N
-	EOF
-
-	cat > /usr/lib/sysctl.d/50-rpi3.conf <<-EOF
-		# Avoid running out of DMA pages for smsc95xx (bsc#1012449)
-		vm.min_free_kbytes = 2048
-	EOF
-fi
-
 #======================================
 # Configure Vagrant specifics
 #--------------------------------------
-if [[ "$kiwi_profiles" == *"Vagrant"* ]]; then
-        # create vagrant user
-        useradd vagrant
-        # allow password-less sudo
-        echo "vagrant ALL=(ALL)NOPASSWD:ALL" > /etc/sudoers.d/vagrant
-        # add vagrant's insecure key
-        mkdir -p /home/vagrant/.ssh
-        chmod 0700 /home/vagrant/.ssh
-        cat > /home/vagrant/.ssh/authorized_keys << EOF
-ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key
-EOF
-        chmod 0600 /home/vagrant/.ssh/authorized_keys
-        chown -R vagrant /home/vagrant
-fi
 
-if [[ "$kiwi_profiles" == *"Ceph"* ]]; then
-  pip install fastapi uvicorn
-  baseInsertService aquarium
-fi
+chmod 0755 /home/vagrant
```

Comment:
We many not want a Vagrant image. I don't think it's a good idea to do vagrant-specific things by default.

Reply:
I can put that into vagrant specific profile but do we need to add other user e.g.live iso and container as well? If that's the case we need some kind of user one way or another.